DEVELOPER.md

```diff
@@ -43,34 +42,30 @@ pip install -r extension_service/requirements-test.txt -r langchain_tools_demo/r
 
 ### Running the frontend
 
-1. [Optional] Set up [Application Default Credentials](https://cloud.google.com/docs/authentication/application-default-credentials#GAC):
-
-    ```bash
-    export GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json
-    ```
-
 1. Change into the demo directory:
 
     ```bash
     cd langchain_tools_demo
     ```
 
-1. Set the server port:
+1. To use a live extension service on Cloud Run:
 
-    ```bash
-    export PORT=9090
-    ```
+    1. Set up [Application Default Credentials][ADC]:
 
-1. [Optional] Set `BASE_URL` environment variable:
+        ```bash
+        export GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json
```

Comment:
We should encourage `gcloud auth application-default login` over a service account key